### ISSUE TITLE
Headings named "undefined"

### ISSUE THREAD

**Issue by Friedolito** [johansatge/obsidian-automatic-table-of-contents#1]
Thank you for this plug-in! This is a function I wished for quite some time. 

Unfortunately it doesn't work for me yet. 
Alle headings are named "undefined"
I used the documented insert. 

![Screenshot -2023-09-25 at 14 21 57](https://github.com/johansatge/obsidian-automatic-table-of-contents/assets/53973194/fcf1a286-13fb-4173-a255-f84beafe70b8)

**Comment by scndthe2nd** [comment]
I'm also getting this error. I believe this has something to do with function on line 126 / line 131 in main.js.

**Comment by scndthe2nd** [comment]
``````

#example_topic_hashtag #example_subject_hashtag


```table-of-contents
style: nestedList # TOC style (nestedList|inlineFirstLevel)
maxLevel: 0 # Include headings up to the speficied level
includeLinks: true # Make headings clickable
debugInConsole: false # Print debug info in Obsidian console
```
# Title

## Chapter N
 - Idea #example_core_principal
 - Concept
 - Important note [[example_date]]

### Item definition
- Concepts around that thing
- Explanation of that thing
	- Further breakdown of that thing #example_related_principal


# Chapter K

# REF
- [[link to reference]]
- [EXT: title thing](<https://example.com>)

``````

**Comment by scndthe2nd** [comment]
My Output

![image](https://github.com/johansatge/obsidian-automatic-table-of-contents/assets/15825472/dfcb6958-e1e3-4ced-aa02-e8d807586cc4)

**Comment by Jack-Chronicle** [comment]
Getting the same error. It works fine when the table is set to not include links however, if that helps narrow it down at all

**Comment by johansatge** [comment]
It did help 👍 issue should be fixed in version `1.0.2` that was pushed a few minutes ago

**Comment by scndthe2nd** [comment]
Looks good! Thanks for resolving this!
